diagnostic.md

```diff
@@ -35,5 +36,5 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{my-contact.my-phone}}
```

Comment:
you have to iterate and assign like so:

{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}